repository-management.md

```diff
@@ -0,0 +1,142 @@
+* `scope:bootstrap` - Part of the bootstrap process, which loads both WP-CLI as well as WordPress Core.
+* `scope:distribution` - Part of the distribution process, where the Phar is being built and releases are produced.
+* `scope:documentation` - Part of the handbook, command reference or inline help.
+* `scope:framework` - Part of the WP-CLI framework itself, which provides the architecture, API and helper functions to make commands possible.
+* `scope:testing` - Part of the unit or functional tests.
+* `scope:website` - Part of the website infrastructure on `wp-cli.org` or `make.wordpress.org`.
+
+#### State
+
+The labels that defined what state a given issue/pull request is in are prefixed with `state:`.
+
+Used states:
+
+* `state:unconfirmed` - The bug/problem in the issue could not be replicated yet or might be related to the reporter's environment.
+* `state:unsupported` - The issue is outside of the scope of a bug report and cannot be supported on GitHub. The reporter should be pointed towards one of the [support channels](http://wp-cli.org/#support).
+
+#### Command
+
+The labels that define what specific command a given issue/pull request is related to are prefix with `command:`.
+
+Some examples:
+
+* `command:core` - Relates to one or more of the subcommands under the `wp core` parent command.
+* `command:cli-check-update` - Relates to the `wp cli check-update` command.
+* `command:post-meta-update` - Relates to the `wp post-meta update` command.
+ 
+### Required Labels
+
+Some labels have a special meaning and/or might be used for automated workflows down the road. These are required across all official packages.
+
+#### Bug
+
+The `bug` label denotes an issue that is a confirmed bug where the code does not produce the expected result.
+
+#### Good First Issue
+
+The `good-first-issue` label denotes an issue that is a good entry point for new contributors wanting to get their feet wet.
+
+Good first issues are small in scope and don't require extensive technical expertise or historical project knowledge to get started.
+
+## Commits
+
+No direct commits are to be done against the `master` branch of the packages. All code changes need to go through a pull request workflow. 
+
+## Pull Requests
+
+All code changes go through a pull request.
+
+Every submitted pull request needs to go through a code review, and needs to be approved by at least one of the [committers](https://github.com/orgs/wp-cli/teams/committers).
+
+Non-trivial pull requests should be preceded by a related issue that defines the problem to solve and allows for discussion of the most appropriate solution before actually writing code.
+  
+If a pull request was submitted by one of the [committers](https://github.com/orgs/wp-cli/teams/committers), the submitter should set the "Reviewers" for that pull request to `wp-cli/committers` if a general code review is needed, or to one or more specific committer profiles if the expertise of a specific person is needed/wanted.
```

Comment:
I'm with @Sidsector9 here, "is submitted" sounds better (to my delicate ears anyway), ["present perfect"](https://en.wikipedia.org/wiki/Uses_of_English_verb_forms#Present_perfect) or somethink...

Reply:
https://github.com/wp-cli/handbook/pull/127/commits/334ee9f4856fd2aea5a1b2b5c58823ef9b6a90f7